src/analysis/retail/shaman/restoration/modules/Abilities.ts

```diff
@@ -564,7 +564,7 @@ class Abilities extends CoreAbilities {
         isUndetectable: true, // its not but, why.
       },
       {
-        spell: SPELLS.FROST_SHOCK.id,
+        spell: TALENTS.LAVA_BURST_TALENT.id,
```

Comment:
another frost shock -> lava burst

Reply:
changed to FROST_SHOCK

Reply:
I removed edits here to avoid confusion, since this PR is for elemental and not resto shaman